src/controllers/BaseController.php

```diff
@@ -14,4 +14,16 @@ public function __construct() {
     public function render($view) {
         echo $view->render();
     }
+    
+    protected function parsePath() {
```

Comment:
@jetbrains-junie make more clear that the controller part is removed already (e.g. "parsePathWithoutController")